layers/+lang/scheme/packages.el

```diff
@@ -102,17 +102,44 @@
         "sR" 'geiser-eval-region-and-go
         "ss" 'geiser-set-scheme)
 
+      (evil-define-key 'insert 'geiser-repl-mode-map
+        (kbd "S-<return>") 'geiser-repl--newline-and-indent)
+
       (evil-define-key 'normal 'geiser-repl-mode-map
-      "gj" 'geiser-repl-next-prompt
-      "gk" 'geiser-repl-previous-prompt)
+        "]]" 'geiser-repl-next-prompt
+        "[[" 'geiser-repl-previous-prompt
+        "gj" 'geiser-repl-next-prompt
+        "gk" 'geiser-repl-previous-prompt)
 
+      (spacemacs/declare-prefix-for-mode 'geiser-repl-mode "mh" "help")
       (spacemacs/set-leader-keys-for-major-mode 'geiser-repl-mode
-        "q" 'geiser-repl-exit
+        "b" 'geiser-pop-symbol-stack
         "c" 'geiser-repl-clear-buffer
         "i" 'geiser-repl-interrupt
```

Comment:
this one should be C-d.
Because, literally, ^D has the meaning of end of transmission.

And this is indeed the default key binding of most repl to signal EOT.

"i" is not really meaningful. I'd more likely to think it represents "insert".

Then insert lambda is "il", and import module is "im".

Reply:
Yes, nice suggestions :+1:

Reply:
It [seems that the `geiser-pop-symbol-stack` is quite useless in the REPL](https://www.nongnu.org/geiser/geiser_4.html#Jumping-around), I put it here because `evil-collection` has it. But I will remove it.

Reply:
```elisp
(defun geiser-edit-symbol-at-point (&optional arg)
  "Opens a new window visiting the definition of the symbol at point.
With prefix, asks for the symbol to edit.")

(defun geiser-pop-symbol-stack ()
  "Pop back to where \\[geiser-edit-symbol-at-point] was last invoked.")
```

I think both functions are not useful...
To jump to the symbol-at-point, you can do `g d` or `SPC m g d`. Then `C-o` brings you back to the original marker.